test

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+#
+# test - a pa test script
+
+failures=0
+fail() {
+    printf "%s\n" "$*"
+    failures=$((failures + 1))
+}
+
+export PA_DIR=test-stuff/passwords
+
+# clean up previous run
+# (the previous state is left around
+# intentionally, in case the dev
+# wants to poke around)
+# that's why we don't clean up on exit
+rm -rf test-stuff
+
+# pa welcomes you
+./pa | grep -q "a simple password manager" ||
+    fail "pa should print a welcome message\n"
```

Comment:
redundant \n

Reply:
good catch. fixed in [06b996a](https://github.com/biox/pa/pull/48/commits/06b996a4d8deb35e90e44afa9b322e5180f2d021)